src/ralph/admin/sitetrees.py

```diff
@@ -123,6 +123,17 @@ def section(section_name, app, model):
             ]
         ),
         ralph_item(
+            title=_('Domains'),
+            url='#',
+            url_as_pattern=False,
+            perms_mode_all=False,
+            children=[
+                section(_('Domain contracts'), 'domains', 'DomainContract'),
+                section(_('Domain registrants'), 'domains', 'DomainRegistrant'),
+                section(_('Domain owners'), 'accounts', 'RalphUser'),
```

Comment:
is this a mistake? what is the purpose of this menu entry, since we have 'Users' already?